frontend/src/scenes/surveys/SurveyTemplates.tsx

```diff
@@ -24,7 +24,7 @@ export function SurveyTemplates(): JSX.Element {
                 title={'New survey'}
                 buttons={
                     <LemonButton type="primary" to={urls.survey('new')} data-attr="new-blank-survey">
-                        Create blank survey
+                        New from blank
```

Comment:
I think it's supposed to be "Create blank survey" for both rather than "New from blank"

Reply:
^ This would be nice to update though

Reply:
I did :)